cns/restserver/api_test.go

```diff
@@ -1292,6 +1292,37 @@ func TestCreateHostNCApipaEndpoint(t *testing.T) {
 	fmt.Printf("createHostNCApipaEndpoint Responded with %+v\n", createHostNCApipaEndpointResponse)
 }
 
+func TestNCIDCaseSensitive(t *testing.T) {
```

Comment:
add comment explaining the test
The test should be testing case insensitive instead of senstive